codecov_auth/views/okta_cloud.py

```diff
@@ -72,7 +72,9 @@ def get(
             )
             return HttpResponse(status=404)
 
-        app_redirect_url = get_app_redirect_url(org_username, service)
+        app_redirect_url = get_app_redirect_url(
+            organization.username, organization.service
+        )
         oauth_redirect_url = get_oauth_redirect_url(org_username, service)
```

Comment:
does the same also make sense for `oauth_redirect_url`?

Reply:
Oh, actually the `oauth_redirect_url` is not using either fields, so I can probably just remove them. Thanks for the catch!